File: apps/blog-service-api-admin/src/api/article/ArticleWhereInput.ts
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { StringFilter } from "../../util/StringFilter";
import { DateTimeNullableFilter } from "../../util/DateTimeNullableFilter";

export type ArticleWhereInput = {
  author?: StringNullableFilter;
  content?: StringNullableFilter;
  id?: StringFilter;
  publishedOn?: DateTimeNullableFilter;
  title?: StringNullableFilter;
};

File: apps/ArticleService/src/APIs/Article/Dtos/ArticleWhereInput.cs
namespace ArticleService.APIs.Dtos;

public class ArticleWhereInput
{
    public string? Author { get; set; }

    public string? Content { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? Id { get; set; }

    public DateTime? PublishedOn { get; set; }

    public string? Title { get; set; }

    public DateTime? UpdatedAt { get; set; }
}
